Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM ubuntu:16.04
-ENV ELIXIR_VERSION=1.9.4-otp-22
+FROM ubuntu:20.04
+ENV ELIXIR_VERSION=1.11.2-otp-22
```

Comment:
I think we need otp-23 here. @ConnorRigby, do oy know for sure what esl-erlang gets installed? We might need to specify the version on the install.